src/Converters.cxx

```diff
@@ -2610,6 +2610,12 @@ static void* PyFunction_AsCPointer(PyObject* pyobject,
 // function pointer. The former is direct, the latter involves a JIT-ed wrapper.
     static PyObject* sWrapperCacheEraser = PyCFunction_New(&gWrapperCacheEraserMethodDef, nullptr);
 
+    // FIXME: avoid string comparisons and parsing
+    std::string true_signature = signature;
+    true_signature.erase(std::remove(true_signature.begin(), true_signature.end(), ' '), true_signature.end());
```

Comment:
Can you give an example what's the type of things new get and we should edit out?

Reply:
I realize we should probably move forward with this PR. We could probably do a better job at the converter comparison level but that's probably good enough to make progress...